.github/actions/metadata/action.yaml

```diff
@@ -39,6 +39,7 @@ runs:
         echo "commit_hash $(git rev-parse --short $GITHUB_SHA)"
         echo "commit_hash=$(git rev-parse --short $GITHUB_SHA)" >> $GITHUB_OUTPUT
         echo "artefact_commit_hash=${{ github.event.pull_request.head.sha }}"
+        echo "Update to (x + 1) i.e. v0.28"
```

Comment:
test code I would think it's good to remove if you have collected evidence already